azure-iot-common/tests/test_asyncio_compat.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import asyncio
+import sys
+from azure.iot.common import asyncio_compat
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def dummy_coroutine():
+    async def coro():
+        return
+
+    return coro
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
+async def test_get_running_loop(mocker, event_loop):
```

Comment:
>test_get_running_loop [](start = 10, length = 21)

suggestion: Tests are easier to understand if the expected behavior as part of the function name.  Otherwise, I can't see what the test is doing without reading the code and thinking through the logic.  If I'm looking at a list of tests, or if I see that this test fails, I have to read through and think through a bunch of code before I can even guess what's going wrong.  I can guess that it tests the get_running_loop function, but so do a bunch of other tests in this file.  What makes this test important?
e.g: 
test_py37_get_running_loop_calls_asyncio_get_running_loop
test_py36_and_older_get_running_loop_calls_asyncio_underscore_get_running_loop
test_py36_and_older_get_running_loop_throws_if_asyncio_underscore_get_running_loop_returns_none
test_py37_create_task_calls_asyncio_create_task
test_py36_and_older_create_task_calls_asyncio_ensure_future
test_py352_and_newer_create_future_calls_event_loop_create_future
test_py351_and_older_create_future_creates_asyncio_future_object

Reply:
I've made some adjustments that are not quiet the same as these, but should help

---
In reply to: [251158481](https://github.com/Azure/azure-iot-sdk-python-preview/pull/43#discussion_r251158481) [](ancestors = 251158481)